### PR TITLE
updating links from fwlink to real link if exists

### PR DESCRIPTION
**Pull request by eddynaka**
updating links from fwlink to real link if exists - contributes to #3391

### CODE REVIEW

File: docs/framework/wpf/advanced/xaml-syntax-in-detail.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ This topic defines the terms that are used to describe the elements of XAML synt
 ## The XAML Language Specification  
  The XAML syntax terminology defined here is also defined or referenced within the XAML language specification. XAML is a language based on XML and follows or expands upon XML structural rules. Some of the terminology is shared from or is based on the terminology commonly used when describing the XML language or the XML document object model.  
   
- For more information about the XAML language specification, download [\[MS-XAML\]](https://go.microsoft.com/fwlink/?LinkId=114525) from the Microsoft Download Center.  
+ For more information about the XAML language specification, download [\[MS-XAML\]](https://docs.microsoft.com/previous-versions/aa468565(v=msdn.10)) from the Microsoft Download Center.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/xaml-syntax-in-detail.md]:
```suggestion
 For more information about the XAML language specification, download [\[MS-XAML\]](http://download.microsoft.com/download/0/A/6/0A6F7755-9AF5-448B-907D-13985ACCF53E/[MS-XAML].pdf) from the Microsoft Download Center.  
```

File: docs/framework/wpf/advanced/xaml-syntax-in-detail.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ This topic defines the terms that are used to describe the elements of XAML synt
 ## The XAML Language Specification  
  The XAML syntax terminology defined here is also defined or referenced within the XAML language specification. XAML is a language based on XML and follows or expands upon XML structural rules. Some of the terminology is shared from or is based on the terminology commonly used when describing the XML language or the XML document object model.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/xaml-syntax-in-detail.md]: something wrong with the script here.

File: docs/framework/wpf/app-development/native-wpf-browser-hosting-support-apis.md [hunk 3]
```diff
@@ -11,10 +11,10 @@ ms.assetid: 82c133a8-d760-45fb-a2b9-3a997537f1d4
...
+ The WPF Active Document server implements several common hosting interfaces, including [IOleObject](/windows/win32/api/oleidl/nn-oleidl-ioleobject), [IOleDocument](/windows/win32/api/docobj/nn-docobj-ioledocument), [IOleInPlaceActiveObject](/windows/win32/api/oleidl/nn-oleidl-ioleinplaceactiveobject), [IPersistMoniker](https://docs.microsoft.com/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms775042(v=vs.85)), [IOleCommandTarget](/windows/win32/api/docobj/nn-docobj-iolecommandtarget). When hosted in the Web Browser control, these interfaces can be queries from the object returned by the [IWebBrowser2::Document](https://docs.microsoft.com/previous-versions//aa752116(v=vs.85)) property.
```
Comment (mairaw) [inline docs/framework/wpf/app-development/native-wpf-browser-hosting-support-apis.md]:
```suggestion
 The WPF Active Document server implements several common hosting interfaces, including [IOleObject](/windows/win32/api/oleidl/nn-oleidl-ioleobject), [IOleDocument](/windows/win32/api/docobj/nn-docobj-ioledocument), [IOleInPlaceActiveObject](/windows/win32/api/oleidl/nn-oleidl-ioleinplaceactiveobject), [IPersistMoniker](https://docs.microsoft.com/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms775042(v=vs.85)), [IOleCommandTarget](/windows/win32/api/docobj/nn-docobj-iolecommandtarget). When hosted in the Web Browser control, these interfaces can be queries from the object returned by the [IWebBrowser2::Document](https://docs.microsoft.com/previous-versions/aa752116(v=vs.85)) property.  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @eddynaka. Left a few comments to be addressed before merging.

File: docs/framework/wpf/advanced/xaml-syntax-in-detail.md (hunk 2 above)
Comment (eddynaka) [reply docs/framework/wpf/advanced/xaml-syntax-in-detail.md]: i will check what happened. lets just ignore

**Review by mairaw — APPROVED** [review]
Thanks @eddynaka!

**Pull request merged**